src/Controller/UserController.php

```diff
@@ -197,6 +251,18 @@ public function deleteUser($userId, UserRepository $repository, EntityManagerInt
         $manager->remove($registeredUser);
         $manager->flush();
 
+        $keyAll = 'user.all?page=' . $request->query->getInt('page', 1);
+        $keyOnce = 'user.once';
+
+        $cacheAll = $this->adapter->getItem($keyAll);
+        $cacheOnce = $this->adapter->getItem($keyOnce);
+
+        if (true === $cacheAll->isHit()){
```

Comment:
Encore une fois, deux fois la même condition.
Sur un remove, tu clear le cache et tu ne le reconstruis pas. Il sera reconstruit au prochain appel du get

Reply:
> Encore une fois, deux fois la même condition.
No 2 different keys

> Sur un remove, tu clear le cache et tu ne le reconstruis pas. Il sera reconstruit au prochain appel du get
Have I not to define the key here ?